sanic/policy.py

```diff
@@ -0,0 +1,13 @@
+import uvloop
+
+_policy = uvloop.EventLoopPolicy()
+
+def get_event_loop():
```

Comment:
I think it would make sense to add this as a property of the sanic app itself. E.g., have `sanic.loop` return the results of `get_event_loop()`.  What do you think? Also, does this `get_event_loop` always return the current, running event loop?